reference/data_solutions.yaml

```diff
@@ -0,0 +1,411 @@
+              enabled: true
+              is_oauth_connected: true
+            - channel_id: 2
+              id: 4
+              name: Segment.com
+              api_key: SEG-1234567890
+              enabled: true
+              data_tag_enabled: true
+            - channel_id: 2
+              id: 6
+              name: Site Verification Tags
+              verification_tag: <meta name="google-site-verification" content="twLOPnFqwX5JK2-RksTZp_QEkfUPPgfGJ0_UHrEKLcY" />
+              enabled: true
+            - channel_id: 0
+              id: 0
+              name: string
+              connection: string
+              enabled: true
+          meta: {}
+    Analytic:
+      type: object
+      properties:
+        data:
+          oneOf:
+            - $ref: '#/components/schemas/AnalyticGoogle'
+            - $ref: '#/components/schemas/AnalyticVisualWebsiteOptimizer'
+            - $ref: '#/components/schemas/AnalyticFacebookPixel'
+            - $ref: '#/components/schemas/AnalyticSegment'
+            - $ref: '#/components/schemas/AnalyticSiteVerificationTags'
+            - $ref: '#/components/schemas/AnalyticAffiliateConversionTracking'
+        meta:
+          type: object
+          properties: {}
+    AnalyticGoogle:
+      type: object
+      title: Google Analytic
+      properties:
+        channel_id:
+          type: integer
+          description: 'Channel ID. If empty, the default Channel ID will be used.'
+          example: 1
+        id:
+          type: integer
+          description: Web Analytic ID.
+          example: 1
+        name:
+          type: string
+          description: Web Analytic title.
+          example: Google Analytics
+        tracking_code:
+          type: string
+          description: Google Analytic tracking code. Required when version is 1
+          example: GA-1234567890
+          nullable: true
+        property_id:
+          type: string
+          description: Google Analytic property ID. Required when version is 2
+          example: GA-1234567890
+          nullable: true
+        enabled:
+          type: boolean
+          description: Flag indicates if Web Analytic is enabled or not.
+        data_tag_enabled:
+          type: boolean
+          description: Flag indicates if data tags are enabled by Google Analytic.
+        version:
+          type: integer
+          description: Google Analytic version.
+          example: 1
+    AnalyticVisualWebsiteOptimizer:
+      type: object
+      title: Visual Website Optimizer Analytic
+      properties:
+        channel_id:
+          type: integer
+          description: 'Channel ID. If empty, the default Channel ID will be used.'
+          example: 1
+        id:
+          type: integer
+          description: Web Analytic ID.
+          example: 2
+        name:
+          type: string
+          description: Web Analytic title.
+          example: Visual Website Optimizer
+        vwo_smartcode:
```

Comment:
Are each of these models going to have property keys specific to the analytics type? When I run the the request now, they all return the following format with a generic `code` key:

``` json
    {
      "id": 1,
      "channel_id": 0,
      "name": "Google Analytics",
      "code": "8908ZTHQGP",
      "enabled": false,
      "data_tag_enabled": true,
      "version": 2,
      "is_oauth_connected": null
    }
```